qark/plugins/task_affinity.py

```diff
@@ -0,0 +1,73 @@
+import sys, os, re
```

Comment:
Split these into multiple lines for better readability and matching Qark standards.

```python
import os
import re
import sys
```